Example/Source/Examples/Interactive/InsertBrickViewController.swift

```diff
@@ -12,7 +12,9 @@ import BrickKit
 private let Section = "Section"
 
 class InsertBrickViewController: BrickApp.BaseBrickController, HasTitle {
-    
+
+    var data: [String] = ["Brick 1"]
```

Comment:
The repeated bricks have the title `let newElement = "BRICK \(data.count + 1)"`. So this should be upper case String